packages/components/src/components/tag-dismissible/tag-dismissible-styles.ts

```diff
@@ -24,22 +25,23 @@ export const getComponentCss = (color: TagDismissibleColor, hasLabel: boolean):
         verticalAlign: 'top',
         outline: addImportantToRule(0),
       },
+      ...hostHiddenStyles,
       button: {
         position: 'relative',
-        minHeight: pxToRemWithUnit(48),
-        padding: `${pxToRemWithUnit(4)} ${pxToRemWithUnit(46)} ${pxToRemWithUnit(4)} ${pxToRemWithUnit(16)}`,
```

Comment:
Wouldn't it be possible to work with flex + gap instead of defining fixed 54px padding?